proto/tasks.proto

```diff
@@ -37,22 +37,11 @@ message TaskUpdate {
     // *is* included in this map, the corresponding value represents a complete
     // snapshot of that task's stats at in the current time window.
     map<uint64, Stats> stats_update = 3;
-    // The IDs of any tasks which have *completed* since the last update.
-    //
-    // If a task's ID is present in this list, that task has terminated. The
-    // client is free to delete any data it has stored for that task if it is no
-    // longer relevant. Task IDs *may* be reused when new tasks are spawned; if
-    // a new task is spawned for the same ID, it will appear in the `new_tasks`
-    // list in a future `TaskUpdate`.
-    //
-    // When a task ID is present in this list, the `stats_update` field of this
-    // `TaskUpdate` will contain the final statistics recorded for that task.
-    repeated common.SpanId completed = 4;
     // The system time when this update was recorded.
     //
     // This is the timestamp any durations in the included `Stats` were
     // calculated relative to.
-    google.protobuf.Timestamp now = 5;
```

Comment:
just a note: when we stabilize the protobufs (e.g. when we publish a release of `console-api` on crates.io), we will want to stop making proto changes like this; instead, we will want to remove fields [by marking their field numbers as `reserved`](https://developers.google.com/protocol-buffers/docs/proto3#updating). for now, this is fine, since we aren't yet guaranteeing interoperability across git hashes.